frontend/js/game/events.js

```
@@ -10,76 +20,84 @@ class Event {
-                            break;
-                        } else if (!number) {
-                            // User doesn't want to use any weapon
-                            player.health = (player.health || 0) + value;
-                            displayWarningMessage(`${this.description}: ${value}`);
-                            break;
-                        } else {
-                            console.log("Please enter a valid number.");
-                        }
+                    player.inventory.items.weapon -= numWeapons;
+                    if (player.inventory.items.weapon <= 0) {
+                        delete player.inventory.items.weapon;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)